packages/sdk/src/types/CharacterStyleTypes.ts

```diff
@@ -1,30 +1,17 @@
-import { Case, Scripting } from './TextStyleTypes';
-import { ColorUsage, ColorUsageUpdate } from './ColorStyleTypes';
+import { Case, Scripting, SelectedTextStyle } from './TextStyleTypes';
+import { ColorUsageUpdate } from './ColorStyleTypes';
 import { Id } from './CommonTypes';
 
-export type CharacterStyle = {
+
+export type CharacterStyle = Omit<SelectedTextStyle, 'hasLocalFormatting'> & {
```

Comment:
Woah didn't know you could do that.

---
I would Prefer to subclass a more abstract interface maybe `TextStyle` or `ITextStyle` or another name.